ui/components/ui/list-item/list-item.component.js

```diff
@@ -33,6 +34,9 @@ export default function ListItem({
         }
       }}
     >
+      {topContent ? (
+        <div className="list-item__top-content">{topContent}</div>
```

Comment:
I think we can use Text for topContent here?
![Screenshot 2023-06-08 at 12 02 17 AM](https://github.com/MetaMask/metamask-extension/assets/39872794/0c03d820-2dfc-4b6e-9ece-339bb94bdbee)